### PR TITLE
Adding Basic Logging

### PR DESCRIPTION
**Pull request by colinmccloskey**
Added error and above logs to a local log file for now, but can stick with just standard out for now too

Also added main (for testing) and logs to git ignore

### CODE REVIEW

File: src/main/resources/log4j2.xml [hunk 1]
```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~
+  ~ Copyright (c) Meta Platforms, Inc. and affiliates.
+  ~
+  ~ This source code is licensed under the MIT license found in the
+  ~ LICENSE file in the root directory of this source tree.
+  -->
+
+<Configuration status="WARN" monitorInterval="30">
```
Comment (hunterjackson) [inline src/main/resources/log4j2.xml:10]: Does this clean up it's own logs? I don't see any log rotating/deleting.

File: .gitignore [hunk 2]
```diff
@@ -35,4 +35,6 @@ build/
 .vscode/
 
 ### Mac OS ###
-.DS_Store
\ No newline at end of file
+.DS_Store
+src/main/java/com/meta/chatbridge/Main.java
```
Comment (hunterjackson) [inline .gitignore:39]: we're going to need this file to be included very soon :)

File: pom.xml [hunk 3]
```diff
@@ -49,10 +49,25 @@
             <artifactId>javalin</artifactId>
             <version>5.6.1</version>
         </dependency>
+<!--        <dependency>-->
+<!--            <groupId>org.slf4j</groupId>-->
+<!--            <artifactId>slf4j-simple</artifactId>-->
+<!--            <version>2.0.7</version>-->
+<!--        </dependency>-->
```
Comment (hunterjackson) [inline pom.xml:56]: Do we not need this? If not lets delete

File: .idea/encodings.xml [hunk 4]
```diff
@@ -2,6 +2,5 @@
 <project version="4">
   <component name="Encoding">
     <file url="file://$PROJECT_DIR$/src/main/java" charset="UTF-8" />
-    <file url="file://$PROJECT_DIR$/src/main/resources" charset="UTF-8" />
```
Comment (hunterjackson) [inline .idea/encodings.xml:5]: ?

**Review by hunterjackson — CHANGES_REQUESTED** [review]
Lets also add an alternative log file for testing that logs to the console only

File: src/main/resources/log4j2.xml [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+        <Property name="LOG_PATTERN">%d{yyyy-MM-dd HH:mm:ss} %-5p %c{1} - %m%n</Property>
+    </Properties>
+
+    <Appenders>
+        <Console name="console" target="SYSTEM_OUT" follow="true">
+            <JsonLayout>
+                <KeyValuePair key="timestamp" value="$${date:yyyy-MM-dd'T'HH:mm:ss.SSSZ}"/>
+            </JsonLayout>
+        </Console>
+        <File name="file" fileName="logs/app.log">
```
Comment (hunterjackson) [inline src/main/resources/log4j2.xml:21]: anyway to set this dynamically or some an env variable or something? lets also pick a default location that isn't relative and makes sense, maybe `/var/log/cp4m`

Comment (hunterjackson) [reply src/main/resources/log4j2.xml:21]: also if the the sub-directories don't exist does this create them?

File: .idea/encodings.xml (hunk 4 above)
Comment (colinmccloskey) [reply .idea/encodings.xml]: This kept auto-deleting, not sure why

**Comment by colinmccloskey** [comment]
Just removing file logging for now - will add in a second commit with a RollingFileAppender with deletion to keep tracks of logs by date and delete old ones

**Pull request merged**